bot.py

```diff
@@ -1828,8 +1828,9 @@ def build_help():
 		wa = db.get_address(user.id)
 	else:
 		wa = db.get_address(user_id)
+		user = db.get_user_by_id(user_id)
```

Comment:
no probs but it could be simplified to use 1 db query

user = db.get_user_by_id(user_id)
wa = user.wallet_address

Reply:
Ok, didn't know it loaded the whole dataset. Will change it